content/sales.rst

```diff
@@ -0,0 +1,152 @@
+
+This will increase the chance of knowledge transfer inside their organization
+during the trial and decrease instances of conducting more than one session.
+
+It will also smooth the process when the customer does the initial
+configuration during the trial.
+
+
+When demo session is agreed ask for the following:
+--------------------------------------------------
+
+* Dates available
+* Times available (and local timezone, so that we know what GoToMeeting
+  toll free / long distance numbers to provide)
+* Number of attendees (max 4 attendees incl 1 organiser and 1 co-organiser in
+  case or max 5 attendees if 1 organiser)
+* Platform
+* Protocols
+* List or criteria of requirements to know what to set up in advance
+
+
+Sending out demo session details
+--------------------------------
+
+* Session is held in the first 10 minutes. After that it may be rescheduled.
+
+* Phone bridge is recommended, if not, use a VOIP headset.
+
+
+Marketing / Promotional Materials
+=================================
+
+Promotions and marketing materials are mainly located in the main website.
+
+It should be as generic and non-technical as possible with links to the
+Documentation for more in-depth / technical information.
+
+To save materials, or to obtain past materials, the `chevah/design` repository
+is available.
+Please ask the team for access to the repository.
+
+
+Announcing a new release to the email list
+------------------------------------------
+
+After the website is updated and News item published, we send a newsletter:
+
+1. Go to Campaigns in Mailchimp.
+
+2. Select 'Replicate' besides 'NEW: ACME Release Announcement'.
+   If it is a security release, use the Security Advisories email list.
+
+3. Select the News Announcements email list.
+
+4. Update the subject and email with the News text used to announce the
+   new release. You can use the text in the News article.
+
+5. Select Send. Before sending the final email, preview first by going
+   to 'Preview and Send' on the top menu. Select 'Send a test email'.
+
+
+Mock ups and designs for the website
+------------------------------------
+
+If a change involves a design or content addition (such as an image carousel
+in JS), it is a good idea to write/mock up the content first before coding.
+
+In this way, you can check to see what type of code work should be done to best
+communicate the content.
+
+Please go to the 'design' repository for sample images and screenshots to use
+and add your own samples.
+
+If raw HTML needs to be used, see if custom directives can be used such as:
+
+.. sourcecode:: bash
+
+    :call_for_action: Ready to install our product?
+    :call_for_action_link: /pricing/?utm_source=client&utm_campaign=clientbtn&utm_medium=btn#id1
+    :call_for_action_button: Ask for a trial
+
+For documentation pages, please do not add raw HTML as the format is designed
+to be converted into multiple other formats.
+
+
+Campaign/Design Outsourcing
```

Comment:
I think that this should also be removed as we are not practicing it

Reply:
I added it since it was info that I was looking for when I first started (ie beginning of this year) looking for freelance contracts and also since I don't know what the general take is to outsource design.  

This can be added later if needed